src/Common/src/System/Collections/Generic/LowLevelStack.cs

```diff
@@ -28,6 +28,9 @@ namespace System.Collections.Generic
     /// Data size is smaller because there will be minimal virtual function table.
     /// Code size is smaller because only functions called will be in the binary.
     /// </summary>
+#if TYPE_LOADER_IMPLEMENTATION
+    [System.Runtime.CompilerServices.ForceDictionaryLookups]
```

Comment:
@davidwrighton 
This feels like the right thing to do, but I am not sure I understand `ForceDictionaryLookups` enough to tell.

Reply:
Yeah, this is fine. The attribute is protecting against a jit stress mode. Don't worry.